time time         # rospy.Time, ros::Time


float64 east        # east [m]
float64 north       # north [m]
float64 yaw         # yaw [rad]

float64 yawrate     # yawrate [rad/s]
float64 vel         # vel[m/s]
